### ISSUE TITLE
FR: More media type support.

### ISSUE THREAD

**Issue by hemanth** [runkitdev/demokit#12]
It would be useful to provide options for `gif` and other video formats.

**Comment by tolmasky** [comment]
Yeah I was thinking about this a bit, every node library out there seems to use ffmpeg, and rely on ffmpeg already being present. If we could somehow have it be part of the install, or do it ourselves, it would be ideal. If not, I feel like there's going to be a ton of confusion as to why the feature doesn't work. Additionally, this may relate to this bug I'm tracking about changing the way we record videos: https://github.com/runkitdev/demokit/issues/4

**Comment by hemanth** [comment]
> If we could somehow have it be part of the install, or do it ourselves, it would be ideal

^ That would be interesting and really beneficial!
